styles.css

```diff
@@ -0,0 +1,669 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald&display=swap');
+
+*,
+*::before,
+*::after {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.AppBar {
+  height: 82px;
+  width: 375px;
+  border-radius: 0;
+  background: #3c3a39;
+}
+
+.ToolBar {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  height: 48px;
+  width: 375px;
+  left: 0;
+  top: 44px;
+  position: absolute;
+  right: 0;
+}
+
+nav {
+  height: 44px;
+  width: 101px;
+  right: 44px;
+  border-radius: 0;
+  padding: 10px 12px 10px 12px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: absolute;
+  top: calc(50% - 44px / 2);
+}
+
+nav img {
+  text-align: end;
+  margin-left: 84px;
+}
+
+.Logo {
+  height: 44px;
+  width: 101px;
+  left: 12px;
+  border-radius: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: absolute;
+  top: calc(50% - 44px / 2);
+}
+
+.Logo a {
+  font-family: "Roboto", sans-serif;
+  font-weight: 900;
+  font-size: 20px;
+  line-height: 34px;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin-left: 5px;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.headline {
+  width: 375px;
+  height: 590px;
+  background: #1c1a19;
+  background-image: url("images/Illustration.png");
+  background-repeat: no-repeat;
+  background-position: right;
+  background-position-y: 230px;
+  margin-bottom: 0;
+  margin-left: 0;
+  left: 0;
+}
```

Comment:
- [ ] As we are still imitating the Figma design and if we compare your design to the Figma one they are not similar, I can see that your content is not fill all the entire page , kindly put `width:100%` to fit all the entire page and also try to change your `heading` to match your design with the Figma one :wink:

Your Design: :point_down: 
![fixcontent](https://user-images.githubusercontent.com/88894525/198013141-43fdfa01-4ea2-43b3-bb40-cce00761a095.png)

Figma Design: 
![figma](https://user-images.githubusercontent.com/88894525/198013217-9dda9a59-3cce-48dd-97ae-0e1ed1b2d7e3.png)

Reply:
- [ ] if I use that width:100% the result will be beyond the 375px it will consider the display block 
- [ ] The problem here is in position that as I have used position: absolute yet I didn't want to use it since am planning a head to work on big screen 
- [ ] but mean while I have used it for the purposes of passing here